apps/web-giddh/src/app/budget-forecasting/budget-forecasting.component.scss

```
@@ -55,3 +53,23 @@
+    }
+    &.default-theme {
+        .forecast-list,
+        .chart-container {
+            background-color: color-mix(in oklch, var(--theme-accent-light) 30%, var(--color-white)) !important;
+        }
+
+        .ai-question-badge {
+            background-color: var(--color-black) !important;
+            color: var(--color-white) !important;
+        }
+    }
```

Stylelint (17.2.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)